### ISSUE TITLE
Decide on a clearer set of labels

### ISSUE THREAD

**Issue by belak** [sorin-ionescu/prezto#1374]
Here is the current list of [labels](https://github.com/sorin-ionescu/prezto/labels):

* Bug
* Discussion
* Documentation
* Duplicate
* Enhancement
* Feature
* Fix
* Idea
* Invalid
* Module
* Rejected
* Revision
* Theme
* Upstream

I think this could be cleaned up a bit to make this a bit simpler to manage.

Here is my new proposed list (along with quick descriptions for each):

* Bug - either a bug in the issue tracker or a fix for a bug in a pr
* Documentation - info about missing documentation
* Duplicate - fairly self explanatory
* Enhancement - an improvement to something which already exists
* Idea - Alternatively we could keep "Discussion" and drop "Idea", however the current usage of the Discussion label seems to be a catch-all which I'm not a huge fan of
* Invalid - not a bug with prezto. Alternatively we could just close these, or have a "Support" label.
* Module - new module
* Theme - new theme
* Upstream - not a bug with prezto, but worth tracking here because it may impact multiple users

**Comment by johnpneumann** [comment]
I would agree that we need better labels. There's not one for a request for help and it would be helpful to have OS specific labels as well. This is largely administrivia, but it'd be helpful to have folks be able to tag folks based on whether or not they knew what they were able to test, etc. The flip side of that is that we don't want 800 labels either. The set of labels that I've become fond of for my projects has been:

- Priority: Critical
- Priority: High
- Priority: Low
- Priority: Medium
- Status: Abandonded
- Status: Accepted
- Status: Available
- Status: Blocked
- Status: Complete
- Status: Documentation Needed
- Status: In Progress
- Status: In Review
- Status: Help Wanted
- Type: Bug
- Type: Code Documentation
- Type: Enhancement
- Type: Feature
- Type: Invalid
- Type: Investigate
- Type: Maintenance
- Type: Question
- Type: User Documentation
- Type: Wontfix

Some are not useful in this context (I just pulled them from the github api real quick), but I personally like that I can see what everything is easily, without having to dig. 

We're also pretty bad about adding labels, so it's one of those things that I'm not sure warrants the work unless we're going to start being really on top of things.
